### PR TITLE
Consume Flight icons from NPM

### PR DESCRIPTION
**Pull request by randallmorey**
This PR updates the previous flight upgrade and consumes icons from the NPM package `@hashicorp/flight-icons`.

### CODE REVIEW

**Comment by amyrlam** [comment]
Following up from meeting with Randall earlier, made https://github.com/hashicorp/flight/pull/119 as a rough starting point for icon usage guidelines. Not necessary to move forward with this PR just linking the related things!

File: yarn.lock [hunk 1]
```diff
@@ -2066,10 +2066,10 @@
   resolved "https://registry.yarnpkg.com/@handlebars/parser/-/parser-1.1.0.tgz#d6dbc7574774b238114582410e8fee0dc3532bdf"
   integrity sha512-rR7tJoSwJ2eooOpYGxGGW95sLq6GXUaS1UtWvN7pei6n2/okYvCGld9vsUTvkl2migxbkszsycwtMf/GEc1k1A==
 
-"@hashicorp/structure-icons@^1.9.0-0":
```
Comment (calcaide) [inline yarn.lock:2069]: That is how a real deprecation looks like. The PR that brings the new stuff, also deletes the old one :raised_hands: :clap:

**Review by calcaide — APPROVED** [review]
LGTM! Terrific job Randy!! :muscle: :raised_hands:

**Review by gsusmi — APPROVED** [review]
LGTM. Ty!

**Review by DhariniJeeva — APPROVED** [review]

**Pull request merged**